test/ok/ok0109_optionalParams.fram

```diff
@@ -0,0 +1,23 @@
+pub data Option A = None | Some of A
+pub method add = (extern dbl_addInt : Int -> Int -> Int) self
+pub method fn (+) = add
+
+let f {?x} = x
+let a = f
+let b = f {x=2}
+
+let g {?x : Int} () = 
+match x with
+| Some n => n + 2
+| None => 42
+end
```

Comment:
We should format tests correctly. This match expression should be indented. The next (in lines 18-21) should be fixed too.